.github/workflows/pr-master.yml

```
@@ -109,3 +111,47 @@ jobs:
+              (
+                github.event_name == 'pull_request' ||
+                (github.event_name == 'pull_request_review' && github.event.review.state == 'approved')
+              )
+            )
+          )
+        )
+      )
+    needs: [test_go, skip_tests]
+    runs-on: ubuntu-latest
+    steps:
+      - run: |
```

actionlint (1.7.12)

[error] "github.event.pull_request.head.ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)